augur/db/export/__init__.py

```diff
@@ -0,0 +1,19 @@
+"""
+Export data from a database.
+"""
+from augur.utils import first_line
+from ..load import DEFAULT_LOADED_METADATA_TABLE
+from .sqlite3 import export as export_sqlite3
+
+
+def register_parser(parent_subparsers):
+    parser = parent_subparsers.add_parser("export", help=first_line(__doc__))
+    parser.add_argument('--sqlite3', required=True, help="SQLite3 database file to export from.")
+    parser.add_argument('--metadata-table', default=DEFAULT_LOADED_METADATA_TABLE, help="Table name in database that contains metadata.")
+    parser.add_argument('--to-metadata', required=True, help="Metadata file to output.")
```

Comment:
Similar to a comment elsewhere, I would expect this argument to be named `--output-metadata` to be consistent with other Augur commands.